### PR TITLE
[MRG] renaming section header and updating link

### PR DESCRIPTION
**Pull request by choldgraf**
This fixes a section header that I missed in the last PR. Also it looks like the docs didn't build the new tutorial HTML. I'm not sure why but it sounds like maybe it's because of the circle cache? Maybe @Eric89GXL has an idea for that one?

### CODE REVIEW

**Comment by Eric89GXL** [comment]
CircleCI is probably just behind

**Comment by Eric89GXL** [comment]
(and was broken for a while)

**Comment by choldgraf** [comment]
gotcha - apparently my tests are failing on circle too ?

File: examples/io/plot_read_epochs.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import mne
 from mne import io
 from mne.datasets import sample
+from matplotlib import pyplot as plt
```
Comment (agramfort) [inline examples/io/plot_read_epochs.py]: why do you need this?

File: examples/decoding/README.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-Decoding / MVPA
----------------
+Decoding / Encoding
+-------------------
 
-Decoding, a.k.a. MVPA or machine learning examples.
+Decoding, encoding, and general machine learning examples.
```
Comment (agramfort) [inline examples/decoding/README.txt]: I would keep the name MVPA for indexing and eventually explicit the acronym

Comment (choldgraf) [inline examples/decoding/README.txt]: you mean google indexing? fair point...if we're interested in search indexing etc, what if we called it "MVPA, decoding, and machine learning"

Comment (choldgraf) [inline examples/decoding/README.txt]: or "MVPA, decoding, and encoding models"

**Comment by agramfort** [comment]
whatever just make sure we have all the keywords

**Comment by choldgraf** [comment]
mmk latest push has keywords...I just went with `Machine Learning (...` and put keywords in there. I think that's the simplest way of doing it. WDYT?

**Comment by Eric89GXL** [comment]
MRG from your end?

I think Machine Learning (...) is fine

**Comment by choldgraf** [comment]
ready to merge whenever from my end

**Comment by Eric89GXL** [comment]
Thanks @choldgraf

**Pull request merged**